src/rastervision/utils/files.py

```diff
@@ -56,6 +57,22 @@ def sync_dir(src_dir, dest_uri, delete=False):
     subprocess.run(command)
 
 
+def _is_raster(uri, s3_test=False):
+
+    if s3_test:
+        uri = uri.replace('s3://', '/vsis3/')
+
+    try:
+
+        rasterio.open(uri)
```

Comment:
My understanding is that the more recent versions of rasterio use the `s3://` prefix - are we using an old version?

Reply:
I'm not sure about rastervision's docker images as I build my own. Unless you're forcefully installing 1.0a12 (or later), you're only on 0.3.

Reply:
OK, we'll test this as part of running through the PR.